WalletWasabi/KeyManagement/KeyManager.cs

```diff
@@ -198,7 +198,7 @@ public IEnumerable<HdPubKey> GetKeys(KeyState? keyState = null, bool? isInternal
 				{
 					return HdPubKeys.Where(x => x.KeyState == keyState);
 				}
-				else if (keyState == null && isInternal != null)
```

Comment:
Redundant. `isInternal` cannot be null at this point, otherwise it would make true any of the previous if conditions.